### PR TITLE
enable title bar drag&drop

### PR DESCRIPTION
**Pull request by wise0704**
+no need to handle theme changes manually; `EventManager.RemoveEventHandler()` is broken anyway
+better to use existing icon from main form instead of creating a new one

### CODE REVIEW

File: FlashDevelop/Docking/TabbedDocument.cs [hunk 1]
```diff
@@ -40,24 +38,14 @@ public CustomFloatWindow(DockPanel dockPanel, DockPane pane, Rectangle bounds)
         private void InitializeComponents()
         {
             FormBorderStyle = FormBorderStyle.Sizable;
-            Icon = new Icon(ResourceHelper.GetStream("FlashDevelopIcon.ico"));
+            Icon = Globals.MainForm.Icon;
             ShowInTaskbar = true;
             Owner = null;
```
Comment (wise0704) [inline FlashDevelop/Docking/TabbedDocument.cs:43]: Why did you need to set Owner to null here?

Comment (Neverbirth) [reply FlashDevelop/Docking/TabbedDocument.cs:43]: I don't remember every downside of Owner, but there were several, like the child form is always above the main form, and IIRC when minimizing the main form the child form was also minimized and there were problems related to focus.

**Comment by Neverbirth** [comment]
IIRC if you don't listen for the Theme change there were problems with the quick find bar when changing the theme.

**Comment by wise0704** [comment]
I see. Maybe it's better to handle theme change event in QuickFind? Since QuickFind is already an IEventHandler. I'll push changes and tell me how that one goes.

Also, note that EventManager.RemoveEventHandler does not work. I have the fixes in my shortcut branch. Maybe I should make a PR for the fixes.

**Comment by Neverbirth** [comment]
I don't think this change is good... it removes built-in Windows capabilities: now dragging to the borders of the screen doesn't dock the forms, and also people expect to be able to maximize by double clicking in the title bar.

The floating docs worked like this PR at the start, but people in my team complained a lot about these two things.

EDIT: Of course I think it's possible to have everything (VS does it for instance), but this PR doesn't enable it.

**Comment by wise0704** [comment]
>dragging to the borders of the screen doesn't dock the forms

Docking as in the Windows left&right docking?

I'll try to see if there is a way to be able to do both... I'll close this for now then.

**Comment by Neverbirth** [comment]
Yes, that docking.
